app/views/entries/index.html.erb

```diff
@@ -28,8 +28,13 @@
             <%= content_tag :td, entry.author, class: 'table__cell' %>
             <%= content_tag :td, entry.published_at.present? ? l(entry.published_at) : t('.published_at_unknown'), class: 'table__cell' %>
             <%= content_tag :td, entry.read?, class: 'table__cell' %>
-            <%= content_tag :td, class: 'table__cell' do %>
-              <%= link_to t('.show'), entry, class: 'table__link' %>
+            <%= content_tag :td, class: 'table__cell table__cell--actions' do %>
+              <%= link_to entry, class: 'table__link', 'aria-label': t('.show') do %>
```

Comment:
Bit of a nitpick, but `aria` can be handled the same way as `data`, so I'd prefer either `aria_label: t()` or `aria: { label: t() }`